plugins/ros/src/components/riScInfo/riScStatus/RiScStatusComponent.tsx

```diff
@@ -277,13 +349,31 @@ export const RiScStatusComponent = ({
           />
         </Box>
       )}
-
       {/* Error */}
       {!selectedRiSc && (
         <Typography paragraph variant="subtitle1">
           {t('rosStatus.statusBadge.error')}
         </Typography>
       )}
+      {lastModifiedDate && daysSinceLastModified && (
+        <Box mt={2} display="flex" gap={1}>
+          {updatedState && IconComponent && (
+            <img
+              src={IconComponent}
+              alt={`${updatedState.replace('_', ' ')} Icon`}
+              height={24}
+              width={24}
+            />
+          )}
```

Comment:
Checking both `updatedState` and `IconComponent` seems a bit overkill. `IconComponent` is defined whenever `updatedState` evaluates to `true`. I think the following, combined with removing `IconComponent` completely, would be better:
```suggestion
          {updatedState && (
            <img
              src={icons[updatedState]}
              alt={`${updatedState.replace('_', ' ')} Icon`}
              height={24}
              width={24}
            />
          )}
```

Reply:
yup